apps/explorer/src/components/NetworkSelector/NetworkSelector.styled.ts

```diff
@@ -82,25 +76,8 @@ export const NetworkLabel = styled.span`
   font-weight: ${({ theme }): string => theme.fontBold};
   letter-spacing: 0.1rem;
 
-  &.görli {
-    background: ${(): string => goerliColor};
-    color: ${({ theme }): string => theme.black};
```

Comment:
The only difference is for Goerli, it used to display text with black color when other networks have white. I don't think this is important enough to make an exception :)

Reply:
One might say it's because of the contrast 
![image](https://github.com/cowprotocol/cowswap/assets/43217/d9172d6b-9b48-454b-930d-8443e52f14ac)

But I'm not the one to say this, I'm fine with this change.